.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Would bug_report and freture_request go in the same issue that would be opened?

Reply:
pick-and-choose! If one really feels the need to use both, I would think that's two separate issues to be opened. Unless they're really that inseparable, somehow.

Reply:
Gotcha!